File: LetsQuiz/src/Question.java

public class Question {

	private String title;
	private String content;
	private int id;
	private String category;
	private char answer;
	private char[] options;
	
	public Question() {}

	public String getTitle() {
		return title;
	}

	public void setTitle(String title) {
		this.title = title;
	}

	public String getContent() {
		return content;
	}

	public void setContent(String content) {
		this.content = content;
	}

	public int getId() {
		return id;
	}

	public void setId(int id) {
		this.id = id;
	}

	public String getCategory() {
		return category;
	}

	public void setCategory(String category) {
		this.category = category;
	}

	public char getAnswer() {
		return answer;
	}

	public void setAnswer(char answer) {
		this.answer = answer;
	}

	public char[] getOptions() {
		return options;
	}

	public void setOptions(char[] options) {
		this.options = options;
	}
	
	
}

File: LetsQuiz_V5/javascript/Question.js

function Question(title,question,questionAnswers,answer,answerOptions,id,isCorrect){
    this.title = title;
    this.question = question;
    this.questionAnswers = questionAnswers;
    this.answer = answer;
    this.answerOptions = answerOptions;
    this.id = id;
    this.isCorrect = isCorrect;
}

//Getters & Setters
function getTitle(){
    return this.title;
}
function getQuestion(){
    return this.question;
}
function getQuestionAnswer(){
    return this.questionAnswers;
}
function getAnswer(){
    return this.answer;
}
function getAnswerOptions(){
    return this.answerOptions;
}
function getId(){
    return this.id;
}
function getIsCorrect(){
    return this.isCorrect;
}
function setTitle(title){
    this.title = title;
}
function setQuestion(question){
    this.question = question;
}
function setQuestionAnswers(questionAnswers){
    this.questionAnswers = questionAnswers;
}
function setAnswer(answer){
    this.answer = answer;
}
function setAnswerOptions(answerOptions){
    this.answerOptions = answerOptions;
}
function setId(id){
    this.id = id;
}
function setIsCorrect(isCorrect){
    this.isCorrect = isCorrect;
}